.gitignore

```diff
@@ -38,4 +38,5 @@ public/api-docs/*
 .vault-password
 /rr
 _temp_graphql_parse_ide_helper_models.php
-/.bash_history
\ No newline at end of file
+/.bash_history
+/.config/psysh/psysh_history
```

Comment:
`**/psysh_history` probably makes the most sense here so that it is ignored no matter where it gets put into the project.